Course/CourseItem.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Course Item.swift
+//  Course
+//
+//  Created by Марк Горкій on 10.10.2022.
+//
+
+import Foundation
+import SwiftUI
+
+struct ItemShape: View{
```

Comment:
The name ItemShare doesn't suite a view, which uses `Text` in it. The shape is Rectangle, Circle and other